cypress/integration/scheduled-events-timeline.spec.ts

```diff
@@ -96,7 +96,8 @@ context('Scheduled events timeline tests', () => {
       cy.contains(currentHourDateTime)
         .parent()
         .should('have.attr', 'style')
-        .and('include', 'background: rgba(');
+        .and('include', 'background: rgb(')
+        .and('include', 'filter: grayscale(0) opacity(0.6)');
```

Comment:
Maybe for the future we could have a support function that checks what counts as element being disabled, in case we want to update the UI color or transparency level then we would not have to change a lot of tests. Just an idea